massa-execution-exports/src/types.rs

```diff
@@ -43,7 +43,7 @@ pub enum SlotExecutionOutput {
     ExecutedSlot(ExecutionOutput),
 
     /// Finalized slot output
-    FinalizedSlot(Slot),
+    FinalizedSlot(ExecutionOutput),
```

Comment:
@aoudiamoncef I'm not too sure if this is the right thing to happen or not: something got mangled along the way when fixing errors.

Reply:
This has been reverted